src/lib/y2network/interface_config_builder.rb

```diff
@@ -129,6 +133,34 @@ def firewall_zone=(value)
       @firewall_zone = value
     end
 
+    # @return [Startmode]
+    def startmode
+      # in future use only @connection_config and just delegate method
+      startmode = Startmode.create(@config["STARTMODE"])
+      startmode.priority = @config["IFPLUGD_PRIORITY"] if startmode.name == "ifplugd"
+      startmode
+    end
+
+    def startmode=(name)
+      mode = Startmode.create(name)
+      # assign only if it is not already this value. This helps with ordering of ifplugd_priority
+      @connection_config.startmode = mode if @connection_config.startmode.name != mode.name
+      @config["STARTMODE"] = mode.name
+    end
+
+    def ifplugd_priority=(value)
```

Comment:
Please, document (at least the @param). Same for `startmode=` and `ifplugd_priority`